infra/concourse/build/developer-tools/Makefile

```diff
@@ -32,6 +33,7 @@ build-image-developer-tools:
 		--build-arg TERRAFORM_VERSION=${TERRAFORM_VERSION} \
```

Comment:
We need a docker image without confusing traces of terraform

Reply:
@ocsig so you need pure bats image?